pkg/scheduler/framework/v1alpha1/framework_test.go

```diff
@@ -848,6 +848,162 @@ func TestFilterPlugins(t *testing.T) {
 	}
 }
 
+func TestBindPlugin(t *testing.T) {
```

Comment:
Should it be `TestBindPlugins`, refer to `TestFilterPlugins`.

Reply:
Yes, it should. I will make the change